app/components/Button/index.jsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React from 'react'
```

Comment:
Just a stylistic thing, but if we use:

```
import React, { Proptypes } from 'react'
```

It saves adding `React` to every Proptype declaration.